projects/plugins/crm/src/automation/class-automation-bootstrap.php

```diff
@@ -244,6 +244,13 @@ protected function register_actions(): void {
 	 * @return void
 	 */
 	protected function register_workflows(): void {
+		$workflow_repository = new Workflow\Workflow_Repository();
+		$workflows           = $workflow_repository->find_all(
+			array(
+				'active' => 1,
```

Comment:
Use `true` (In the repo we are casting it), if we restore the boolean type in the Workflow model.